src/utils/articleTextSize.js

```diff
@@ -16,50 +14,29 @@ const set = size => {
   localStorage.setItem(KEY, size)
 }
 
-const adjustElement = step => {
-  const elements = document.querySelectorAll(SELECTORS)
-  Array.from(elements).forEach(element => {
-    const adjust = parseFloat(element.getAttribute(FONT_SIZE_ATTRIBUTE)) + get() + step
-    element.style.setProperty('font-size', `${adjust}px`)
-  })
-}
-
 const adjust = step => {
   const newSize = get() + step
   if (newSize < MIN_SIZE || newSize > MAX_SIZE) {
-    return
+    return get()
   }
 
-  adjustElement(step)
   set(newSize)
-}
-
-const reset = () => {
-  adjust(-get() + DEFAULT_SIZE)
-  set(DEFAULT_SIZE)
-}
-
-const init = () => {
-  const elements = document.querySelectorAll(SELECTORS)
-  Array.from(elements).forEach(element => {
-    if (!element.hasAttribute(FONT_SIZE_ATTRIBUTE)) {
-      const fontSize = parseFloat(window.getComputedStyle(element).getPropertyValue('font-size'))
-      element.setAttribute(FONT_SIZE_ATTRIBUTE, fontSize)
-    }
-  })
-
-  adjust(0)
+  return newSize
 }
 
 const getSoftkeyEffect = (onAdjust = () => {}) => {
-  const onKeyArrowLeft = () => { adjust(-1); onAdjust(get()) }
-  const onKeyArrowRight = () => { adjust(1); onAdjust(get()) }
+  const onKeyArrowLeft = () => { adjust(-1); onAdjust(getFontSizeClassName()) }
```

Comment:
For consistency, Softkey functions should be defined in the component that needs them. They used to be shared, that's why they are here, but that's not the case anymore.